velox/exec/Operator.h

```diff
@@ -454,7 +454,9 @@ class Operator : public BaseRuntimeStatWriter {
   }
 
   /// Returns a list of identity projections, e.g. columns that are projected
-  /// as-is possibly after applying a filter.
+  /// as-is possibly after applying a filter. It is used to allow dynamic
```

Comment:
"It is used ..." -> "Used to allow pushdown of dynamic filters generated by HashProbe into the TableScan."

Reply:
Done.